build/packaging/linux/bin/kend

```diff
@@ -256,7 +356,7 @@ case "$1" in
         restart
         ;;
     *)
-        echo "Usages: kend {start|stop|restart|status}"
+        echo "Usages: kend {start|start-docker|stop|restart|status}"
```

Comment:
Do we need `restart-docker`?
```suggestion
        echo "Usages: kend {start|start-docker|stop|restart|restart-docker|status}"
```

Reply:
@KimKyungup I don't think we need restart-docker so far. Using kubernetes, If there is a problem, docker image itself restarts. But we may need it later, depending on docker users needs.